rasa/configurations/autoconfig.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from typing import Text, Dict, Any, List
+
+from rasa.constants import CONFIG_AUTOCONFIGURABLE_KEYS
+from rasa.utils import io as io_utils
+
+logger = logging.getLogger(__name__)
+
+
+# In future iterations, this function and the function `create_config_for_keys` below
+# will get additional input parameters to allow for smarter configuration. E.g.
+# training_data: TrainingData
+# domain: Domain
+# stories: StoryGraph
+def get_autoconfiguration(config):
+    autoconfig_keys = CONFIG_AUTOCONFIGURABLE_KEYS
+    missing_keys = [k for k in autoconfig_keys if k not in config]
+
+    create_config_for_keys(config, missing_keys)
+
+    return config
+
+
+def create_config_for_keys(config: Dict[Text, Any], keys: List[Text]) -> None:
+    import pkg_resources
+
+    if keys:
+        logger.debug(
+            f"The provided configuration does not contain the key(s) {keys}. "
+            f"Running automatic configuration for them now."
+        )
+
+    default_config_path = pkg_resources.resource_filename(
+        __name__, "default_config.yml"
+    )
```

Comment:
we use that a couple of times else where as well. We should move that to a utils function

Reply:
not sure what exactly and how... It's basically a one-liner, and calling a utils function would be a one-liner, too